src/css/App.css

```diff
@@ -4,43 +4,193 @@ a {
 
 .centre-text {
   text-align: center;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+}
+
+.content {
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.content a {
+  color: #fff;
+}
+
+.content a:hover {
+  color: #fd79a8;
+}
+
+.left-text {
+  text-align: left;
+  width: 80% !important;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+}
+
+.gap-div {
+  height: 4rem;
 }
 
 .Menu a {
   color: #fff;
+  text-align: right !important;
 }
 
 .Menu a:hover {
   color: #fd79a8;
 }
 
+.navbar {
+  align-items: right;
+  text-align: right !important;
+}
+
+.sosBtn a {
+  background-color: #fd79a8;
+  color: #343a40!important;
+  font-weight: bold;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.sosBtn a:hover {
```

Comment:
I love this hover thing. 
Worried abt the colours for now cause they're not v high contrast

Reply:
Colours are not finalized, I'm just working on layout. We can change colours literally whenever we want